### PR TITLE
CI tests against Julia 0.6, macOS, and Windows

### PR DESCRIPTION
**Pull request by omus**
Appveyor will need to be enabled for this package. Probably one of the maintainers should enable it. I can add the badges to the README here once I know the Appveyor URL to use.

### CODE REVIEW

**Comment by c42f** [comment]
We should definitely enable this, unfortunately I don't seem to have the permissions to add it under the JuliaGeo organization.  @yeesian @visr - can one of you guys give me the appveyor permission, or just add Geodesy?

**Comment by omus** [comment]
I missed that StaticArrays v0.5.0 is Julia 0.6 only

**Comment by omus** [comment]
Changes worked locally on 0.5 and 0.6. Should be good to go if the CI passes.

**Comment by omus** [comment]
I'll note that the AppVeyor configuration won't currently allow failures for nightly

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +1,7 @@
 julia 0.5-
 CoordinateTransformations 0.3.0
-StaticArrays 0.0.4
+
+# Enforce that:
+# - Julia 0.6 only using StaticArrays 0.5.0 and up
+# - Julia 0.5 uses StaticArrays between 0.0.4 and 0.4.0-
+StaticArrays 0.0.4 0.4.0- 0.5.0
```
Comment (c42f) [inline REQUIRE]: So, does this strange looking version specification allows `Geodesy` version resolution to work on both 0.5 and 0.6?

That's great if it works!  I had an uneasy but vague memory that it was more complicated than this... @andyferris do you remember the issue where we discussed this at length?  There's https://github.com/JuliaArrays/StaticArrays.jl/issues/152 , but that doesn't seem to be it.

Comment (omus) [inline REQUIRE]: It's not pretty but should get the job done. If we really want to support 0.4.0 of StaticArrays we could make the conditional check `Pkg.installed("StaticArrays")` versus using `VERSION`. I ended up doing things this way as the `Pkg.installed` calls isn't as performant as `VERSION`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
-| [![0.4 package tests](http://pkg.julialang.org/badges/Geodesy_0.4.svg)](http://pkg.julialang.org/?pkg=Geodesy) [![0.5 package tests](http://pkg.julialang.org/badges/Geodesy_0.5.svg)](http://pkg.julialang.org/?pkg=Geodesy) | [![master build](https://travis-ci.org/JuliaGeo/Geodesy.jl.svg?branch=master)](https://travis-ci.org/JuliaGeo/Geodesy.jl) [![master coverage](http://img.shields.io/coveralls/JuliaGeo/Geodesy.jl.svg)](https://coveralls.io/r/JuliaGeo/Geodesy.jl) |
+| [![0.5 package tests](http://pkg.julialang.org/badges/Geodesy_0.5.svg)](http://pkg.julialang.org/?pkg=Geodesy) [![0.6 package tests](http://pkg.julialang.org/badges/Geodesy_0.6.svg)](http://pkg.julialang.org/?pkg=Geodesy) | [![Linux/macOS](https://travis-ci.org/JuliaGeo/Geodesy.jl.svg?branch=master)](https://travis-ci.org/JuliaGeo/Geodesy.jl) [![Windows](https://ci.appveyor.com/api/projects/status/github/NgYeeSian/Geodesy.jl?svg=true)](https://ci.appveyor.com/project/NgYeeSian/geodesy-jl) [![Coverage](http://img.shields.io/coveralls/JuliaGeo/Geodesy.jl.svg)](https://coveralls.io/r/JuliaGeo/Geodesy.jl) |
```
Comment (omus) [inline README.md]: It's hard to see but I removed the 0.4 package status badge. Should I keep this badge?

Comment (c42f) [inline README.md]: Nah, I'm not bothered about a badge for 0.4 at this stage.

**Comment by c42f** [comment]
Thanks a lot for doing this :-)

**Comment by visr** [comment]
@c42f since GitHub removed the Admin team it seems I was the only one who could still add projects since I set up the account. That should be fixed now, could you check?

**Comment by c42f** [comment]
Oh, I think I merged this a tad early - the badges come from CI running under @yeesian's account (thanks :) ). @visr thanks for changing the config.  Your changes seem to have worked, and I've now enabled Geodesy over at the JuliaGeo instead:

https://ci.appveyor.com/project/JuliaGeo/geodesy-jl

**Pull request merged**